src/sentry/workflow_engine/models/action.py

```diff
@@ -76,3 +79,15 @@ def trigger(self, job: WorkflowJob, detector: Detector) -> None:
         # get the handler for the action type
         handler = self.get_handler()
         handler.execute(job, self, detector)
+
+
+@receiver(pre_save, sender=Action)
+def enforce_config_schema(sender, instance: Action, **kwargs):
+    """
+    Add a default receiver that
```

Comment:
Is there more to this thought?

Reply:
😅 adhd in code 🤣 --- i'll just remove the comment - i don't think this is anything too hard to grok.